chat-plugins/room-events.js

```diff
@@ -104,8 +107,9 @@ exports.commands = {
 	},
 	roomeventshelp: [
 		`/roomevents - Displays a list of upcoming room-specific events.`,
-		`/roomevents add [event name] | [event date/time] | [event description] - Adds a room event. Requires: # & ~`,
-		`/roomevents remove [event name] - Deletes an event. Requires: # & ~`,
+		`/roomevents add OR create OR new [event name] | [event date/time] | [event description] - Adds a room event. Requires: # & ~`,
```

Comment:
Don't document aliases!!!!

Reply:
There's only one exception: For commonly-used commands, you can include both the default name of the command, and an abbreviation to make it easier to type.

Reply:
👍